### ISSUE TITLE
create_newcase fails for multi-instance case using python3

### ISSUE THREAD

**Issue by johnsonb-ucar** [ESMCI/cime#3766]
${CIMEROOT}/scripts/lib/CIME/case/case.py fails with the following error when attempting to create a multi-instance case when using python3.

```
if multi_driver and ninst>1:
TypeError: '>' not supported between instances of 'str' and 'int'
ERROR: Case could not be created.
```

This can be fixed by using explicit type conversion to convert ninst to an integer. This solution has been implemented in pull request #3765.

**Comment by billsacks** [comment]
It looks like this issue should be closed, given #3765 . @johnsonb-ucar do you agree?

**Comment by johnsonb-ucar** [comment]
Hi Bill: Yes I think it should be closed since the issue is fixed.
